fe2-android/app/src/test/java/com/github/dedis/student20_pop/model/ElectionTest.java

```diff
@@ -14,29 +13,20 @@
 
     private String name = "my election name";
     private String id = "my election id";
-    private String question = "my question";
+    private List<String> questions = Arrays.asList("my questions");
     private boolean writeIn = false;
     private long startTime = 0;
     private long endTime = 1;
     private long creationTime = 0;
-    private List<String> ballotOptions = Arrays.asList("candidate1", "candidate2");
+    private List<List<String>> ballotsOptions = Arrays.asList(Arrays.asList("candidate1", "candidate2"));
     private Election election = new Election();
 
     @Test
     public void settingNullParametersThrowsException() {
         assertThrows(IllegalArgumentException.class, () -> election.setName(null));
-        assertThrows(IllegalArgumentException.class, () -> election.setBallotOptions(null));
```

Comment:
Instead of deleting all these tests you can move them to a file dedicated to test the class `ElectionQuestion` :)

Reply:
There already are tests for ElectionQuestion (that I didn't write) :)